packages/expo-mesh-gradient/expo-module.config.json

```
@@ -1,6 +1,9 @@
 {
-  "platforms": ["apple"],
+  "platforms": ["apple", "android"],
```

```suggestion
  "platforms": ["apple", "android"],
  "coreFeatures": ["swiftui", "compose"],
```
This field is required on Android; otherwise, the compose will not be added.